docs/install.md

```diff
@@ -16,16 +16,15 @@ You can run Compose on OS X and 64-bit Linux.  It is currently not supported on
 the Windows operating system. To install Compose, you'll need to install Docker
 first.
 
-Depending on how your system is configured, you may require `sudo` access to
-install Compose. If your system requires `sudo`, you will receive "Permission
-denied" errors when installing Compose. If this is the case for you, preface the
-install commands with `sudo` to install.
+> Note: If you get a "Permission denied" error, your `/usr/local/bin` directory
+probably isn't writable and you'll need to install Compose as the superuser. Run
+`sudo -i`, then the two commands above, then `exit`.
```

Comment:
Hm, I don't see "the two commands above"; should this note be moved somewhere else?